packages/typeorm/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) IBM Corp. 2017,2019.
+Node module: @loopback/repository
```

Comment:
Inconsistent node module name header.

Ditto for the other module name headers.

Reply:
Haha, yea need to clean up.